exporter/opentelemetry-exporter-prometheus-remote-write/proto/generate-proto-py.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
can you add some directions for running this script? E.g. it looks like it needs to be run from this directory

Also instead of adding the proto files to this repo directly, could you make this script download and patch them? It's easier to avoid any copyright/licensing confusion and make it easier to update if new proto files are released.

You might be able to re-use some of this https://github.com/open-telemetry/opentelemetry-python/blob/main/scripts/proto_codegen.sh

Reply:
That's what I had in mind, SGTM!

Reply:
LGTM, can we just delete these proto files which are pulled in from the script after generation or add them to .gitignore?